packer/ansible/roles/node/tasks/main.yml

```diff
@@ -1,15 +1,21 @@
 ---
-- name: Download n Installer
-  get_url: url=http://git.io/n-install dest=/tmp/n-install mode=0755
-  register: download_n
-  sudo: yes
 
-- name: Install n
-  shell: "yes | N_PREFIX={{ ansible_env.HOME }}/n /tmp/n-install"
-  sudo: yes
-  when: download_n|changed
+# source ~/.bashrc is for interactive shell
+- name: Install nvm and Appending source string to ~/.bashrc
+  shell: wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.31.0/install.sh | bash
```

Comment:
Yeah the problems I've seen before, and that seem apparent here, is that the ansible script only installs nvm for the vagrant user? But when you run on-http/on-dhcp-proxy/on-tftp with root as is required, they will not run the nvm installed version of node.